src/js/Tabs/__test__/addItem.test.ts

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+import Tabs from '../index';
+
+const messages = {
+  INVALID_ARGUMENT: 'Error: invalid function arguments',
+  MISSING_TAB_NAME: 'Missing tab name on tab item[{{index}}]',
+  INVALID_ACTION: 'Behavior invalid',
+  MISSING_NEW_ITEM_TABNAME: 'Missing tab name.'
+};
+
+describe('Unit test Tabs addItem', () => {
+  beforeEach(() => {
+    jest.spyOn(console, 'error');
+    // @ts-ignore
+    console.error.mockImplementation(() => {});
+  });
+  afterEach(() => {
+    // @ts-ignore
+    console.error.mockRestore();
+  });
+
+  test('addItem is called correctly', () => {
+    const newTabName = 'Tab3';
+    const items = [{
+      tabName: 'Tab1',
+    }, {
+      tabName: 'Tab2',
+    }];
+    const myTabs = new Tabs({items});
+    const newTab = {
+      tabName: newTabName,
+    };
+    myTabs.addItem(newTab);
+    const tabItems = myTabs.getItems();
+    expect(tabItems).toBeTruthy();
```

Comment:
This line is redundent

Reply:
This line is to ensure `tabItems` existed before verifying `length` of `tabItems`